src/pages/cart/CartPage.tsx

```diff
@@ -1,115 +1,24 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import { Composition, Box, Only, query } from 'atomic-layout'
+import { Composition, Box } from 'atomic-layout'
 import styled from 'styled-components'
-import { IoMdClose, IoIosArrowForward } from 'react-icons/io'
-import { Heading } from '../../atoms/Heading'
+import { IoIosArrowForward } from 'react-icons/io'
 import { Grid } from '../../atoms/Grid'
+import { CartItemsList } from './components/CartItemsList'
+import { useSelector } from 'react-redux'
```

Comment:
Can you please arrange your imports, so that you have third-party packages coming first, and then your own imports following them?

```js
import a from 'a' // this is a third-party file (i.e library)
import b from 'b'

import c from './c' // pay attention: it's your _local_ file
import d from './d'
```